test/byContract/IexecPocoBoost/IexecPocoBoost.test.ts

```diff
@@ -142,15 +141,17 @@ describe('IexecPocoBoost', function () {
         oracleConsumerInstance = await new TestClient__factory()
             .connect(anyone)
             .deploy()
-            .then((contract) => contract.deployed());
+            .then((contract) => contract.waitForDeployment());
         gasWasterClientInstance = await new GasWasterClient__factory()
             .connect(anyone)
             .deploy()
-            .then((contract) => contract.deployed());
+            .then((contract) => contract.waitForDeployment());
+        gasWasterClientAddress = await gasWasterClientInstance.getAddress();
         someContractInstance = await new OwnableMock__factory()
             .connect(anyone)
             .deploy()
-            .then((contract) => contract.deployed()); // any other deployed contract would be fine
+            .then((contract) => contract.waitForDeployment()); // any other deployed contract would be fine
+        someContractAddress = await someContractInstance.getAddress();
```

Comment:
```suggestion
            .then((contract) => contract.waitForDeployment())
            .then((deployedContract) => deployedContract.getAddress());
```
`someContractInstance` can be renamed by `someContractAddress`